pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -338,6 +351,13 @@ public EncryptionKeyInfo getPrivateKey(String keyName, Map<String, String> keyMe
             consumerBuilder.cryptoKeyReader(keyReader);
         }
 
+        if (arguments.maxNumMessages > 0 || arguments.maxNumBytes > 0 || arguments.timeout > 0) {
+            consumerBuilder.batchReceivePolicy(BatchReceivePolicy.builder()
+                    .maxNumMessages(arguments.maxNumMessages)
+                    .maxNumBytes(arguments.maxNumBytes)
+                    .timeout(arguments.timeout, TimeUnit.MILLISECONDS).build());
+        }
+
```

Comment:
Only set the batchReceivePolicy does not able to measure the performance of batch-receive. Batch recieve is a separate method.